aleemer/Internify
# [IN-89] Redirect Create page to View page when creating JobPosting on last step

#### User Story
[[IN-89] Redirect Create page to View page when creating JobPosting on last step](https://chaosneutral.atlassian.net/jira/software/projects/IN/boards/1?selectedIssue=IN-89)

#### Changes made
- Added redirect in `Review` page to go to `View` page when job posting is added
- Added new `User` typed redux action to update the user store with the new job posting to fix overwriting current user store
- Added bulk endpoint call `/api/jobs/bulk` that takes in an array of `jobId`s and returns all the job postings from the database
- Added new `jobId` field to `TableBasic` so that the table in the `Profile` page is able to redirect to the `View` page
- Added new `dateCreated` field in `JobPostingSchema` to keep track of when the job posting was created

#### Does your new code introduce new warnings on dev console?
It shouldn't have. It was fine on my machine but I'm getting inconsistent results from a variety of scenarios. Double check on your machine.

#### Test Steps
1. Login
2. Create a job posting
3. Once you've reached the `Review` step, click 'Create' to add the job posting
4. You should be redirected to the respective `View` page with URI `/view/:jobId`
5. Click on the back button on the top left of the `View` page and you should be redirected to the `Profile` page
6. You should see the newly updated list of jobs in the table on the right of the `Profile` page
